pkg/controller/ecs/service/setup.go

```diff
@@ -80,6 +80,12 @@ func postObserve(_ context.Context, cr *svcapitypes.Service, resp *svcsdk.Descri
 		return obs, err
 	}
 
+	if resp.Services[0].DesiredCount != resp.Services[0].RunningCount {
+		obs.ResourceUpToDate = true
```

Comment:
Is it necessary to set this? What if the tasks are not ready yet because of some misconfiguration? If we hardcode `obs.ResourceUpToDate = true` here users will never be able to update their configuration if there are no tasks running.

I think we should remove setting this here unless it is really necessary.

Reply:
I don't think it's strictly necessary. I have update the PR accordingly.